tutorials/spectrum_pipe.ipynb

```diff
@@ -96,7 +96,7 @@
     "    reference=1 * u.TeV,\n",
     ")\n",
     "\n",
-    "flux_point_binning = EnergyBounds.equal_log_spacing(0.7, 30, 5, u.TeV)"
+    "flux_point_binning = EnergyBounds.equal_log_spacing(0.77, 30, 5, u.TeV)"
```

Comment:
why the change?

Reply:
This change was not intended, I'll revert it.